airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/CdcAcceptanceTests.java

```diff
@@ -594,6 +595,7 @@ private WebBackendConnectionUpdate getUpdateInput(final ConnectionRead connectio
   }
 
   // can be helpful for debugging
+  @SuppressWarnings("PMD")
```

Comment:
I know this is already merged but could we just suppress the specific PMD failure instead of all of PMD?